student.rb

```diff
@@ -1,6 +1,8 @@
 require './person'
 
 class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] When setting the classroom for a student it also adds it to the classrooms' students:

- [ ] Check for a method that serves as the 'belongs-to' side (a student belongs to a classroom)

Something like:

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
Done